### PR TITLE
Implement the "maps" and "continents" endpoints

### PR DESCRIPTION
**Pull request by fanonwue**
As a continuation of #7, I finally had some time to work on this again. I think I have everything under the `/maps` and `/continents` endpoints covered now, I just need to add some more tests.

### CODE REVIEW

File: src/main/kotlin/com/kryszak/gwatlin/clients/mapinfo/MapInfoClient.kt [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        )
+
+    fun getTasks(continentId: Int, floorId: Int, regionId: Int, mapId: Int) =
+        getRequest<List<Int>>(
+            "$continentsEndpoint/$continentId/floors/$floorId/regions/$regionId/maps/$mapId/tasks"
+        )
+
+
+    private inline fun <reified R : Any> handleOneOrMultipleIds(ids: Collection<Int>, lang: String, endpoint: String): List<R> {
+        if (ids.isEmpty()) return emptyList()
```
Comment (Kryszak) [inline src/main/kotlin/com/kryszak/gwatlin/clients/mapinfo/MapInfoClient.kt:61]: This function could be defined with `when` construct like:
```
return when {
   ids.isEmpty() -> emptyList()
   ids.size == 1 -> listOf(getRequest("$endpoint/${ids.first()}?lang=$lang")) 
   else -> getRequest("$endpoint?ids=${ids.joinToString(",")}&lang=$lang")
}
```

File: src/main/kotlin/com/kryszak/gwatlin/serializers/PairSerializer.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.kryszak.gwatlin.serializers
+
+import com.google.gson.*
+import java.lang.reflect.Type
+
+class PairSerializer: JsonSerializer<Pair<Any, Any>> {
+    override fun serialize(src: Pair<Any, Any>?, typeOfSrc: Type?, context: JsonSerializationContext): JsonElement {
+        if (src == null) return JsonNull.INSTANCE
```
Comment (Kryszak) [inline src/main/kotlin/com/kryszak/gwatlin/serializers/PairSerializer.kt:8]: Kotlin has nullable operators for this kind of stuff, like:
```
return src?.let {
        val jsonArray = JsonArray()
        jsonArray.add(context.serialize(it.first))
        jsonArray.add(context.serialize(it.second))
        return jsonArray
} ?: JsonNull.INSTANCE
```

**Comment by Kryszak** [comment]
Also: please increase project version in `build.gradle` to 1.5, so we can release Map info functionality right after this PR is merged.

**Comment by fanonwue** [comment]
The PR should be ready now. I've included the improvements you proposed and added some more documentation. It seems like the build is failing now when uploading the coverage report to Codacy, that might be related to the switch from Codecov?

I'm also thinking about whether it would be better to use the "Accept-Language" header instead of the "lang" query parameter to specify the language, since we already use a header to specify the requested schema version. Maybe you have an opinion on this?

**Comment by Kryszak** [comment]
Regarding build issue it seems that it fails because PR is coming from forked repo that do not have proper secret set required by Codacy step (CODACY_PROJECT_TOKEN). As it build properly at master we can ignore that.

Regarding language: we can switch to header for consistency, I'll create an issue for that.

Code looks good - I'll be merging it into master soon.

**Pull request merged**